playbooks/init/base_packages.yml

```diff
@@ -35,13 +35,16 @@
         name: "{{ pkg_list | join(',') }}"
```

Comment:
I have thought about this, I don't wish to implement that at this time.  I just want this one small fix to ship, we'll have to start evaluating other built-in vars such as major_version and it will be a little messy to flesh all that out here.